### PR TITLE
chore: adjust start cmd

### PR DESCRIPTION
**Pull request by fireairforce**
`pnpm run tsc -- --watch` was wrong, everytimes i adjust a package will get error, so i replace all here...

### CODE REVIEW

File: packages/core-loggers/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   "repository": "https://github.com/pnpm/pnpm/blob/master/packages/core-loggers",
   "scripts": {
-    "start": "pnpm run tsc -- --watch",
+    "start": "pnpm tsc --watch",
```
Comment (zkochan) [inline packages/core-loggers/package.json]: it should be just `tsc --watch`

there's no need to start an additional pnpm process.

Comment (fireairforce) [reply packages/core-loggers/package.json]: replace all cmd here?

Comment (zkochan) [reply packages/core-loggers/package.json]: yes

Comment (fireairforce) [reply packages/core-loggers/package.json]: done

**Review by zkochan — APPROVED** [review]

**Comment by fireairforce** [comment]
feel free to merge it, i need this to adjust my code at local env in other branch.😂 thank u~ @zkochan

**Pull request merged**